src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1095,10 +1095,10 @@ private void AddOrUpdateWakeupTimer(string databaseName, IdleDatabaseActivity id
             // in case the DueTime is negative or zero, the callback will be called immediately and database will be loaded.
             
             _wakeupTimers.AddOrUpdate(databaseName,
```

Comment:
don't you need to get the AddOrUpdate result and materialize it?

Reply:
done